dev/com.ibm.ws.security/src/com/ibm/ws/security/context/SubjectManager.java

```diff
@@ -87,7 +87,7 @@ public void clearSubjects() {
      * Gets the subject thread context that is unique per thread.
      * If/when a common thread storage framework is supplied, then this method
      * implementation may need to be updated to take it into consideration.
-     * 
+     *
```

Comment:
Probably can remove this file as you didn't really make any logical changes to it.